app/models/user.rb

```diff
@@ -1,8 +1,12 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable
+
```

Comment:
- [ ] Good job adding devise to your Blog application, however as requested by the previous code reviewer, you should make sure that your User model contains the `confirmable` so that whenever the user signup , he should receive the confirmation email 😄 , Kindly add that to meet the requirements 👍. For more insight about this subject kindly follow this [link](https://www.youtube.com/watch?v=-PaxytvhscE)

Reply:
Thank you @iambenkis for the comment, you are Right. This change wasn't committed but he has been now.